fix: build fails after `mvn clean`

The reported error is

```sh
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.439 s
[INFO] Finished at: 2025-02-10T18:14:13+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project schema-service: Compilation failure: Compilation failure: 
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[34,9] Symbol nicht gefunden
[ERROR]   Symbol: Variable log
[ERROR]   Ort: Klasse com.materialidentity.schemaservice.GlobalExceptionHandler
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[35,17] Symbol nicht gefunden
[ERROR]   Symbol: Methode setMessage(java.lang.String)
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[37,55] Symbol nicht gefunden
[ERROR]   Symbol: Methode getStatus()
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[44,9] Symbol nicht gefunden
[ERROR]   Symbol: Variable log
[ERROR]   Ort: Klasse com.materialidentity.schemaservice.GlobalExceptionHandler
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[45,17] Symbol nicht gefunden
[ERROR]   Symbol: Methode setMessage(java.lang.String)
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[46,55] Symbol nicht gefunden
[ERROR]   Symbol: Methode getStatus()
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[53,9] Symbol nicht gefunden
[ERROR]   Symbol: Variable log
[ERROR]   Ort: Klasse com.materialidentity.schemaservice.GlobalExceptionHandler
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[54,17] Symbol nicht gefunden
[ERROR]   Symbol: Methode setMessage(java.lang.String)
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[55,55] Symbol nicht gefunden
[ERROR]   Symbol: Methode getStatus()
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[62,9] Symbol nicht gefunden
[ERROR]   Symbol: Variable log
[ERROR]   Ort: Klasse com.materialidentity.schemaservice.GlobalExceptionHandler
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[63,17] Symbol nicht gefunden
[ERROR]   Symbol: Methode setMessage(java.lang.String)
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[64,55] Symbol nicht gefunden
[ERROR]   Symbol: Methode getStatus()
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[71,9] Symbol nicht gefunden
[ERROR]   Symbol: Variable log
[ERROR]   Ort: Klasse com.materialidentity.schemaservice.GlobalExceptionHandler
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[72,17] Symbol nicht gefunden
[ERROR]   Symbol: Methode setMessage(java.lang.String)
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[73,55] Symbol nicht gefunden
[ERROR]   Symbol: Methode getStatus()
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[80,9] Symbol nicht gefunden
[ERROR]   Symbol: Variable log
[ERROR]   Ort: Klasse com.materialidentity.schemaservice.GlobalExceptionHandler
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[81,17] Symbol nicht gefunden
[ERROR]   Symbol: Methode setMessage(java.lang.String)
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[82,55] Symbol nicht gefunden
[ERROR]   Symbol: Methode getStatus()
[ERROR]   Ort: Variable apiError von Typ com.materialidentity.schemaservice.apiexception.ApiError
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/GlobalExceptionHandler.java:[93,9] Symbol nicht gefunden
[ERROR]   Symbol: Variable log
[ERROR]   Ort: Klasse com.materialidentity.schemaservice.GlobalExceptionHandler
[ERROR] /Users/hannes/Workspace/s1seven/schemas/src/main/java/com/materialidentity/schemaservice/PDFBuilder.java:[58,46] Symbol nicht gefunden
[ERROR]   Symbol: Methode getSource()
[ERROR]   Ort: Variable xsltTransformer von Typ com.materialidentity.schemaservice.XsltTransformer
[ERROR] -> [Help 1]
[ERROR] 
```

--- a/pom.xml
+++ b/pom.xml
@@ -294,6 +294,19 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <annotationProcessorPaths>
+            <path>
+              <groupId>org.projectlombok</groupId>
+              <artifactId>lombok</artifactId>
+              <version>1.18.32</version>
+            </path>
+          </annotationProcessorPaths>
+        </configuration>
+      </plugin>
     </plugins>
   </build>
 </project>

